.kokoro/nightly/samples.cfg

```diff
@@ -36,3 +36,9 @@ env_vars: {
   key: "ENABLE_BUILD_COP"
   value: "true"
 }
+
+# Tell the trampoline which tests to run.
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/java-bigquery/.kokoro/run_samples_resource_cleanup.sh"
```

Comment:
Do you want this to still run the samples tests? This will only run the cleanup script.

Reply:
those are run in the [java8-samples](https://github.com/googleapis/java-bigquery/blob/b1bbddcb87ef5742949e4ec069a4a49b12f9536a/.kokoro/nightly/java8-samples.cfg#L29) and [java11-samples](https://github.com/googleapis/java-bigquery/blob/master/.kokoro/nightly/java11-samples.cfg#L29) configs.

Reply:
Thank you for the reminder -- excluded from synth.py.